src/Illuminate/Session/Middleware/StartSession.php

```diff
@@ -44,11 +47,53 @@ public function handle($request, Closure $next)
             return $next($request);
         }
 
+        $session = $this->getSession($request);
+
+        if ($this->manager->shouldBlock()) {
+            return $this->handleRequestWhileBlocking($request, $session, $next);
+        } else {
```

Comment:
`else` could be removed

Reply:
Laravel usually keeps them for readability.